packages/renderer/src/api/Renderer.ts

```diff
@@ -0,0 +1,10 @@
+import Component from './Component';
+import ComponentsMap from './ComponentsMap';
+
+export default interface Renderer {
+  randomPrefix: string;
+
+  renderLayouts(componentsMap: ComponentsMap): void;
+  renderItems(componentsMap: ComponentsMap): void;
+  renderItem(nodeId: string, component: Component): void;
```

Comment:
Can you please explain the use case when we will need this method publicly?

Reply:
`renderItem` will be used by canvas in the future because you know if we have 3 tabs, only one tab is active, another tab is empty, you can't render Items in not existed place.

canvas will trigger `renderItem` and renderer should render item in the new active tab.

`renderLayouts` - will render layouts
`renderItems ` - will render items by ids <div id="bla"></div>
`renderItem` - will render item that tmp not in layout but can be available in the future.